### ISSUE TITLE
Add option to initialize nullables, instead of setting them to null

### ISSUE THREAD

**Issue by elibosley** [zhouzi/graphql-codegen-factories#62]
It would be awesome if you could merge this fork: https://github.com/immuta/graphql-codegen-factories (not my code) in order to allow nullable values to be treated as non-null when generating mocks. This would greatly improve the usefulness of this library for me.

Thanks a ton, awesome work on this!

**Comment by zhouzi** [comment]
Hey! I'm happy to bring in any improvement 👍

I just had a look at the fork but it seems to implement a few different things so I am not sure to identify the changes you are interested in.

Would you like for all nullable fields to be treated as non-nullable from the schema? Should it also be reflected in the types or only in the factories' return value? 

So given the following schema:

```graphql
type Post {
  id: ID
  title: String
  excerpt: String
}
```

It should generate:

```typescript
interface Post {
  id: string;
  title: string;
  excerpt: string;
}

function createPostMock(props: Partial<Post>): Post {
  return {
    id: "",
    title: "",
    excerpt: "",
    ...props,
  };
}
```

Or the fields should remain nullable in the type but filled with default values in the factory:

```typescript
interface Post {
  id: string | null;
  title: string | null;
  excerpt: string | null;
}

function createPostMock(props: Partial<Post>): Post {
  return { 
    id: "",
    title: "",
    excerpt: "",
    ...props,
  };
}
```

**Comment by elibosley** [comment]
Hey! The first thing you discussed would be amazing - so basically add a flag to treat nullables as non-nullable if you request - allowing generation of non-nullable data. I know this is a problem with my schema but it would really help me for testing and I think would be a great QOL improvement!

**Comment by zhouzi** [comment]
Thanks for the suggestion, I can definitely see how that would be useful. It should be straightforward to implement, I'll look into it 👍

**Comment by zhouzi** [comment]
@elibosley I just released v1.2.0 with this feature. See the documentation for an example of how you can configure it: https://gabinaureche.com/graphql-codegen-factories/configuration/schema#configmaybevaluedefault

Let me know if you have any feedback 🙏
